### DISCUSSION TITLE
Compression?

### DISCUSSION THREAD

**Discussion by SgiobairOg** [beginner-corp/begin-community#69] [Q&A] [answered]
Hi all, just working with a small Vue app and looking at lighthouse scores. My Vue app generates Brotli and Gzip files on build but those files aren't being served for my static folder.

Is there configuration that will ask this of the underlaying AWS structure? Or maybe something I'm misunderstanding on serving compressed files?

**Comment by brianleroux** [comment, marked as answer]
Hey sorry for the delayed response! Yes we have Cloudfront CDN URLs on the settings page. Those should be compressed. Lmk if this works for you; may need to tweak Vue build settings for prefixing with URLs.

**Reply by SgiobairOg** [reply]
Thanks Brian, I'll have a look
